src/controller/response/stripe-response.ts

```diff
@@ -31,20 +31,22 @@ export interface StripePaymentIntentResponse extends BaseResponse {
   clientSecret: string;
 }
 
+// TODO find a fix for integer enums.
```

Comment:
Please add a reference to the bug report you created at the `express-jsdoc-swagger` repository.